spec/memery_spec.rb

```diff
@@ -119,6 +119,16 @@ class F
   def m; end
 end
 
+class G
+  include Memery
+
+  def self.macro(name)
+    define_method(:macro_received) { name }
+  end
+
+  macro memoize def g; end
+end
```

Comment:
1. It can be tested like `expect(G.memoize(:g)).to eq :g`
2. Also via class-reader and `@memoize_result = memoize def d`, I guess.

Reply:
Good call, but i want to avoid G modifications in specs.

I.e. i could use F, but when i call memoize on F, it will have `F#m` memoized

Reply:
> but i want to avoid G modifications in specs.

`define_method` is modification, but OK.

> I.e. i could use F, but when i call memoize on F, it will have F#m memoized

That's why stabbing consts are better then global definition.

Reply:
> define_method is modification, but OK.

Is is called before suite

> That's why stabbing consts is better then global definition.

This PR does not point specs refactoring, but i agree with that a lot

Reply:
> This PR does not point specs refactoring

I know, I know, just side notes.